src/windows/cpu.rs

```diff
@@ -4,30 +4,38 @@ use crate::sys::tools::KeyHandler;
 use crate::{CpuExt, CpuRefreshKind, LoadAvg};
 
 use std::collections::HashMap;
+use std::ffi::c_void;
 use std::io::Error;
 use std::mem;
 use std::ops::DerefMut;
-use std::ptr::null_mut;
 use std::sync::Mutex;
 
-use ntapi::ntpoapi::PROCESSOR_POWER_INFORMATION;
-
-use winapi::shared::minwindef::FALSE;
-use winapi::shared::winerror::{ERROR_INSUFFICIENT_BUFFER, ERROR_SUCCESS};
-use winapi::um::handleapi::CloseHandle;
-use winapi::um::pdh::{
+use windows::core::{s, PCSTR, PCWSTR};
+use windows::Win32::Foundation::{
+    CloseHandle, BOOLEAN, ERROR_INSUFFICIENT_BUFFER, ERROR_SUCCESS, FALSE, HANDLE,
+};
+use windows::Win32::System::Performance::{
     PdhAddEnglishCounterA, PdhAddEnglishCounterW, PdhCloseQuery, PdhCollectQueryData,
     PdhCollectQueryDataEx, PdhGetFormattedCounterValue, PdhOpenQueryA, PdhRemoveCounter,
-    PDH_FMT_COUNTERVALUE, PDH_FMT_DOUBLE, PDH_HCOUNTER, PDH_HQUERY,
+    PDH_FMT_COUNTERVALUE, PDH_FMT_DOUBLE,
+};
+use windows::Win32::System::Power::{
+    CallNtPowerInformation, ProcessorInformation, PROCESSOR_POWER_INFORMATION,
+};
+use windows::Win32::System::SystemInformation::{
+    GetLogicalProcessorInformationEx, RelationAll, RelationProcessorCore, SYSTEM_INFO,
+    SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX,
+};
+use windows::Win32::System::SystemInformation::{
```

Comment:
Instead of this huge import, could you just import `SystemInformation` please? Then just use `SystemInformation::...`.